store/frontend_test.go

```diff
@@ -82,3 +83,25 @@ func TestSetMemoryMBAndCpu(t *testing.T) {
 		})
 	}
 }
+
+func TestNewFrontendsFile(t *testing.T) {
+	// since there is no dummy yml file on git repo, skipping test on Travis can be reasonable...
+	tst.SkipCI(t)
+	definedMemory :=uint(4096)
+	definedCPU := 2.0
+	frontendstore, err := NewFrontendsFile("test_ymls/frontend_test.yml")
```

Comment:
Suggestion: change to a temporary file that you delete in the test?

Reply:
👍🏻